Formula/yarn.rb

```diff
@@ -3,8 +3,8 @@
 class Yarn < Formula
   desc "Javascript package manager"
   homepage "https://yarnpkg.com/"
-  url "https://registry.npmjs.org/yarn/-/yarn-0.15.1.tgz"
-  sha256 "f99fd587e84987909d5f9e918b8fe524349fdc548e5bc5c380c8f8c0a70c6b87"
+  url "https://yarnpkg.com/downloads/0.15.0/yarn-v0.15.0.tar.gz"
```

Comment:
@MikeMcQuaid What do you mean by "It'll make it easier for us to query for new versions"? The URL format is basically the same. We don't want to tightly-couple Yarn releases with GitHub's system, hence the yarnpkg.com URL.

You can get the latest Yarn version number from https://yarnpkg.com/latest-version, would that help in any way?

> It may be worth using the auto-generated GitHub tarball from the tag if that's going to make that easier

This won't work as it has the source code, not the compiled version.